include/pybind11/detail/common.h

```diff
@@ -751,7 +761,7 @@ class PYBIND11_EXPORT builtin_exception : public std::runtime_error {
 #endif
 
 #define PYBIND11_RUNTIME_EXCEPTION(name, type) \
-    class PYBIND11_EXPORT name : public builtin_exception { public: \
+    class PYBIND11_EXPORT_EXCEPTION name : public builtin_exception { public: \
```

Comment:
Does line 753 (above) also need to be changed?

Similarly, line 7 in tests/test_exceptions.h?

Reply:
Yes they need.